--- a/cmake/packaging/linux.cmake
--- b/cmake/packaging/linux.cmake
@@ -17,17 +17,12 @@
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sunshine.service"
             DESTINATION "${SUNSHINE_ASSETS_DIR}/systemd/user")
 else()
-    find_package(Systemd)
     find_package(Udev)
 
     if(UDEV_FOUND)
         install(FILES "${SUNSHINE_SOURCE_ASSETS_DIR}/linux/misc/60-sunshine.rules"
                 DESTINATION "${UDEV_RULES_INSTALL_DIR}")
     endif()
-    if(SYSTEMD_FOUND)
-        install(FILES "${CMAKE_CURRENT_BINARY_DIR}/sunshine.service"
-                DESTINATION "${SYSTEMD_USER_UNIT_INSTALL_DIR}")
-    endif()
 endif()
 
 # Post install
